web/docs/connectors/tcp.md

```diff
@@ -0,0 +1,77 @@
+# tcp
+
+Loads bytes from a TCP or TLS connection.
+
+## Synopsis
+
+```
+tcp [--listen] [-k] [--tls] [--certfile] [--keyfile] <uri>
```

Comment:
We should streamline this with the `zmq` connector, which uses `--connect` and `--bind`. Either this becomes `--bind` or `zmq` should be `--listen`.

Reply:
Hm, `--bind` as a boolean flag doesn't make sense to me, an argument called `--bind` should have an address as argument. Using `-l`/`-k` also has the advantage that the interface is analogous to `nc`, which is a good mental model to have for this connector.

So I tend towards renaming the `zmq` option to `--listen`, but it probably makes sense to have this in a separate PR?